consensus/src/round_manager.rs

```diff
@@ -1118,6 +1136,8 @@ impl RoundManager {
                         "[RoundManager] Unable to process the created QC {:?}",
                         qc
                     ))?;
+                // Question: Should we broadcast order vote or fast share first?
+                self.broadcast_fast_shares(qc.certified_block()).await;
```

Comment:
We should feature gate this as well.
If order votes enabled, only `broadcast_fast_shares` when sending order votes; otherwise, only `broadcast_fast_shares` here.

Reply:
Moved the statement behind the order_vote_enabled feature flag.